lib/sda/overlay.go

```diff
@@ -28,17 +28,26 @@ type Overlay struct {
 	entityListLock sync.Mutex
 	// cache for relating token(~Node) to TreeNode
 	cache TreeNodeCache
+
+	// TreeNodeInstance part
+	instances     map[TokenID]*TreeNodeInstance
+	instancesInfo map[TokenID]bool
```

Comment:
I don't think we are talking about the same thing. This map is just to store which instances are `Done` so when the messages are received, we don't create a new protocol if its destination is an already `Done` PI. It's not about the config or whatever, so `instancesSentInfo` is misleading for me. If you want to change name, what do you say about `instancesDone` ?